app/views/docs/service.phtml

```diff
@@ -174,7 +174,7 @@ $paramSDK = (!empty($sdk)) ? '?sdk='.$sdk : '';
             </div>
             <div class="col span-4 clear">
                 <form class="pull-end desktops-only" onchange="this.submit()"
-                      method="get" action="/docs/<?php echo $this->escape($family); ?>/<?php echo $serviceName; ?>">
+                      method="get" data-ls-attrs="action=/docs/<?php echo $this->escape($family); ?>/<?php echo $serviceName; ?>{{router.hash}}">
```

Comment:
Same about the XSS injection.

Reply:
Seems like the hash is sanitized a bit such that double quotes are converted to `%22` so you wouldn't be able to add an additional HTML attribute to execute JS:

<img width="1082" alt="image" src="https://user-images.githubusercontent.com/1477010/197590046-aca0a523-7088-481c-96f4-2decf8659aa0.png">